### ISSUE TITLE
Padding is invalid

### ISSUE THREAD

**Issue by Niteowlpc** [basildane/WakeOnLAN#66]
The last week or so I have been getting this error message when I go to shut down machines. All I have to do is hit ok and it still shuts down the machine, but that gets a bit tedious when I am shutting down 30 machines and that error boxes keeps stacking on top of each other waiting for me to hit the ok button.
![capture](https://cloud.githubusercontent.com/assets/3299603/25920380/c64f347a-359f-11e7-8467-27e86be8a7bd.PNG)

**Comment by basildane** [comment]
Please try this.  Pick one machine that is having problems, go to the Shutdown tab and reenter the password.  Then try it.

Did you recently update the software to 2.11.16?

**Comment by Niteowlpc** [comment]
Yes, we recently upgraded to 2.11.16. I do not use any username or password on the shutdown tab, I just let it use the default credentials that I am currently logged into. If it means anything, I get the same error just by right clicking on a host and choosing "Properties" from the menu.
![capture2](https://cloud.githubusercontent.com/assets/3299603/25923998/72140fc0-35ae-11e7-860b-39d04b70586d.PNG)

**Comment by basildane** [comment]
OK, can you still go to the page and make the password empty, then click OK.
That should reset the password error.

**Comment by Niteowlpc** [comment]
I dont use any username or password on the shut down page, so its already blank.

**Comment by Niteowlpc** [comment]
Although none of them have a username or password on the shut down page, some do have a domain name on them, could that cause it? 
![capture3](https://cloud.githubusercontent.com/assets/3299603/25924996/cfc2b9b4-35b3-11e7-9d90-169e3167a337.PNG)

**Comment by basildane** [comment]
No, it's not the domain.  I know your password is already blank.
You must hit OK in order for it rewrite the password value to the database.

**Comment by Niteowlpc** [comment]
I am sorry, I misunderstood what you wanted me to do. That fixed it. I simply went into each one that was giving the error, went to the shut down page and hit ok. Now its not doing it anymore.

**Comment by basildane** [comment]
Great.  Sorry about the trouble.  It was a one-time problem going to version .16 and the encryption algorithm changed.  I didn't handle the upgrade cleanly.
